src/inspector_js_api.cc

```diff
@@ -259,6 +259,16 @@ void Open(const FunctionCallbackInfo<Value>& args) {
     agent->WaitForConnect();
 }
 
+void WaitForConnection(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Agent* agent = env->inspector_agent();
+  if (!agent->IsActive()) {
+    env->ThrowError("inspector error, inspector.open should be called first");
```

Comment:
Can this check be made from JavaScript code? I believe the guideline was not to throw exceptions from the native code.